README.adoc

```diff
@@ -22,7 +27,9 @@ Provision a database quickly with https://sandbox.neo4j.com/?usecase=movies[Neo4
 
 === Setup
 
-First get yourself setup with link:http://docs.python-guide.org/en/latest/dev/virtualenvs/[virtualenv] so we don't break any other Python stuff you have on your machine. After you've got that installed let's setup an environment for our app:
+Install Python 3.6-3.9 (sync) or 3.7-3.10 (async).
+
+Then get yourself setup with link:http://docs.python-guide.org/en/latest/dev/virtualenvs/[virtualenv] so we don't break any other Python stuff you have on your machine. After you've got that installed let's setup an environment for our app:
```

Comment:
```suggestion
Then get yourself set up with link:http://docs.python-guide.org/en/latest/dev/virtualenvs/[virtualenv] so we don't break any other Python stuff you have on your machine. After you've got that installed let's set up an environment for our app:
```

Reply:
:D was there before. Good you caught it now.